ddtrace/tracer/sqlcomment.go

```diff
@@ -141,6 +148,9 @@ func commentQuery(query string, tags map[string]string) string {
 	if query == "" {
 		return b.String()
 	}
+	if log.DebugEnabled() {
```

Comment:
I don't think you need to check if DebugEnabled, you can just call log.Debug and it will not log if debug is disabled

Reply:
Yeah, I was wondering about that. The comment on that DebugEnabled said it was for things in "hot paths" to avoid the interface{} allocations. I wasn't sure if this would be considered a hot path or not. I'm fine to just switch to log.Debug.

Reply:
Updated ✅